bin/main.js

```diff
@@ -111,7 +111,18 @@ export async function run(argv) {
       // eslint-disable-next-line no-console
       console.log('');
       log.error('Problems encountered:');
-      errors.forEach(e => log.error(e));
+      console.log('');
+      errors.forEach((e, i, l) => {
+        log.error(e.message ? e.message.toString() : e.toString());
```

Comment:
I now log the message property if present.

I log the entire error when passed `--debug`